crawl-ref/source/dat/database/randname.txt

```diff
@@ -1118,7 +1168,7 @@ jewelled
 
 ichor-stained
 
-distressingly furry
+surprisingly fuzzy
```

Comment:
```suggestion
distressingly furry
```

This one is a crawl classic!!!

Reply:
it is! but when i've introduced new players to crawl, two or three different people have said something along the lines of "what's this? random subtle furry hate in this game?"